lib/constants/ranks.ts

```diff
@@ -31,16 +31,29 @@ export enum Ranks {
   Feast = 'FEAST',
 
   /**
-   * Memorials are either obligatory or optional; their observance is integrated into
-   * the celebration of the occurring weekday in accordance with the norms set forth in the
-   * General Instruction of the Roman Missal and of the Liturgy of the Hours.
-   * Obligatory Memorials which fall on weekdays of Lent may only be celebrated as
-   * Optional Memorials.
-   * If several Optional Memorials are inscribed in the Calendar on the same day, only
-   * one may be celebrated, the others being omitted. (UNLY #14)
+   * Obligatory Memorials are liturgical commemorations of saints, events, or aspects of the faith.
```

Comment:
In general, `obligatory memorial` (and any other celebration rank) is not a proper noun in English, therefore, it should not be written with capital initial letters, unless it is part of a title (heading, celebration name, etc).

We might want to move this to a separate PR, as this should be fixed in all files and I have no problem with doing so. Therefore, consider this comment as non-blocking. :wink:

```suggestion
   * Obligatory memorials are liturgical commemorations of saints, events, or aspects of the faith.
```

Reply:
I like the convention of capitalizing these names to emphasize the importance of these names, which are the subject of the description here. This writing rule should only be used in such cases (for describing a property or an acronym, for example).

However, at the same time, and for the sake of consistency, I understand your point and your preference for following the general writing rules.

Before choosing one writing style over the other, what are your thoughts on this?

Reply:
> to emphasize the importance of these names

Based on what you what to accomplish, I’d choose one of the following:
- reference a type: use [TSDoc links](https://tsdoc.org/pages/tags/link/). e.g. `{@link Ranks.OptionalMemorial}` (once per TSDoc block);
- emphasise some strings: use Markdown syntax for bold and/or italics;
- emphasise some code: use Markdown syntax for inline code snippets.

I’d always use Markdown syntax in TSDocs anyway. I’d also try to use grammatically correct code comments in simple English.

> acronym

I think we need to create a glossary or a list of terms where we would define some of the most commonly used stuff, including acronyms and shortenings (which would link to an `entry with full name via `see $link`). Then, we could use `{@links $link}` in TSDocs when needed.

Reply:
Okay, and thank you for your recommendations. I will update these descriptions following your suggestions 👍

Reply:
> I think we need to create a glossary or a list of terms where we would define some of the most commonly used stuff

Indeed, this is clearly something I would like to see added to Romcal.